src/busy-indicator.scss

```diff
@@ -3,6 +3,7 @@
 @import "./mixins";
 
 $block: #{$fd-namespace}-busy-indicator;
+$message-toast: #{$fd-namespace}-message-toast;
```

Comment:
doesn't this go against the `only one top level fd-* class is used in the file` rule?

Reply:
Hmmm I think I might have a better idea. Let me test it and will let you know.

Reply:
ok, I just pushed, should be available soon in the preview. Thanks for pushing me to think more.
I added more specificity by introducing additional modifier class. Now if you want to use only busy indicator with a label, without a container you need `fd-busy-indicator-extended` wrapper. It will center the content. When you use it with a message toast you will need `fd-message-toast fd-busy-indicator-extended fd-busy-indicator-extended--message-toast`. This is more specific and overwrites the rules set by Message Toast. In future if we need the same scenario for Dialog we can introduce a new modifier class.

Reply:
Thank you for the changes, looks good! Just one minor comment, will approve it meanwhile.